### PR TITLE
fix(dup-drafts): de-dup article-id from drafts

### PR DESCRIPTION
**Pull request by tx0c**

### CODE REVIEW

File: src/queries/article/user/articles.ts [hunk 1]
```diff
@@ -27,12 +29,14 @@ const resolver: UserToArticlesResolver = async (
     inRangeEnd: input.filter?.inRangeEnd,
   })
 
-  return connectionFromPromisedArray(
-    draftService.dataloader.loadMany(
+  const drafts = uniqBy(
```
Comment (robertu7) [inline src/queries/article/user/articles.ts:32]: It won't work if duplicated articles are across two pages?

File: src/queries/user/recommendation/newest.ts [hunk 2]
```diff
@@ -44,10 +44,10 @@ export const newest: RecommendationToNewestResolver = async (
     .as('newest')
 
   const [_countRecord, articles] = await Promise.all([
-    MAX_ITEM_COUNT, // knex.select().from(query.clone().limit(MAX_ITEM_COUNT)).count().first(),
+    MAX_ITEM_COUNT, // always 500 // knex.select().from(query.clone().limit(MAX_ITEM_COUNT)).count().first(),
     knex
       .select()
-      .from(query.clone().limit(MAX_ITEM_COUNT))
```
Comment (robertu7) [inline src/queries/user/recommendation/newest.ts:50]: Remember why we used `clone' here?

**Review by robertu7 — APPROVED** [review]

File: src/queries/article/user/articles.ts (hunk 1 above)
Comment (tx0c) [reply src/queries/article/user/articles.ts:32]: still need `.distinctOn`

File: src/queries/user/recommendation/newest.ts (hunk 2 above)
Comment (tx0c) [reply src/queries/user/recommendation/newest.ts:50]: the common pattern of

```js
const [ articlesCount, articles ] = Promise.all([
  500, // query.count(),
  query.clone().orderBy(...)...
])
```

no longer need the `.clone()` when the other clause is turning to const 500;

**Pull request merged**